scripts/generate_enums.py

```diff
@@ -0,0 +1,356 @@
+#! /usr/bin/env python3
+import argparse
+import os
+import string
+import textwrap
+import xml.etree.ElementTree as ElementTree
+
+CONFIG_FILE_PATH = "../build/docs/public_api/xml/config_8h.xml"
+
+ROC_JAVA_BASE_PATH = "../../roc-java"
+ROC_GO_BASE_PATH = "../../roc-go"
+
+ROC_JAVA_PACKAGE = "org.rocstreaming.roctoolkit"
+
+ODD_PREFIXES = {'roc_clock_source': 'ROC_CLOCK_', 'roc_protocol': 'ROC_PROTO_'}
+
+
+def to_camel_case(name):
+    return ''.join(x.capitalize() for x in name.split('_'))
+
+
+class DocComment:
```

Comment:
nit: consider using namedtuple or dataclass for such types

Reply:
dataclass 👍